docs/cli/edgedb_analyze.rst

```diff
@@ -9,23 +9,31 @@ edgedb analyze
 
     This CLI feature is compatible with EdgeDB server 3.0 and above.
 
+.. note::
+
+    This feature is also available inside the EdgeDB REPL and UI in which
```

Comment:
Yes, and!

I think we should also maybe add a section somewhere on https://www.edgedb.com/docs/cli/edgedb to talk about this feature and update this to be a link to that documentation.

Reply:
I see the page does mention the command but makes it look like it's backslash only so changed that. Also added the video plus another while I was at it, and a quick something else I noticed where a classic Windows command wasn't specified.

Reply:
What do you think about making the text "EdgeDB REPL" a link to the appropriate place in the REPL docs?

Reply:
I would like to have documentation of the UI, but so far, the most we have is the documentation of the `edgedb ui` command which doesn't really tell you anything about the UI except how to invoke it.

We have also made a distinction between the CLI REPL and the UI REPL, and we might want to keep with that here. `analyze` works across the CLI REPL, the UI REPL, and the UI's query editor.